src/Neo/Plugins/Plugin.cs

```diff
@@ -102,20 +103,8 @@ private static void ConfigWatcher_Changed(object sender, FileSystemEventArgs e)
             switch (GetExtension(e.Name))
             {
                 case ".json":
-                    try
-                    {
-                        Plugins.FirstOrDefault(p => p.ConfigFile == e.FullPath)?.Configure();
-                    }
-                    catch (FormatException) { }
-                    break;
                 case ".dll":
-                    if (e.ChangeType != WatcherChangeTypes.Created) return;
-                    if (GetDirectoryName(GetDirectoryName(e.FullPath)) != PluginsDirectory) return;
-                    try
-                    {
-                        LoadPlugin(Assembly.Load(File.ReadAllBytes(e.FullPath)));
-                    }
-                    catch { }
+                    Utility.Log(nameof(Plugin), LogLevel.Warning, $"File {e.Name} is {e.ChangeType}, please restart node.");
```

Comment:
If multiple files are added or deleted, many logs will be printed.

Reply:
Multi log is normal, better than multi loading. Delete all is not a normal behaviour when node is running.